src/utils/session.ts

```diff
@@ -1,9 +1,32 @@
-const TOKEN_NAME = 'token';
```

Comment:
this file is fine, but at wolox, we usually have this at service/localStotage.service.ts